app/views/shared/_footer.html.erb

```diff
@@ -1,6 +1,22 @@
 <footer class="bg-white dark:bg-gray-900" aria-labelledby="footer-heading">
   <h2 id="footer-heading" class="sr-only">Footer</h2>
-  <div class="mx-auto max-w-7xl px-6 pb-8 pt-16 sm:pt-24 lg:px-8">
+  <% if not current_page?(support_us_path) and not current_page?(about_path) %>
```

Comment:
Nit: Our linter should have picked this up, but I think it gave it a pass in the view. There are some quirks with the precedence of the `not` & `and` keywords. `Unless` is probably the best alternative:

```suggestion
  <% unless current_page?(support_us_path) || current_page?(about_path) %>
```